test/script_tools.py

```diff
@@ -52,7 +60,7 @@ def test_threshold_default_setting(self, mock_proceed, mock_lock, mock_lockfile)
         mock_proceed.return_value = True
         mock_lockfile.return_value = mock.MagicMock()
 
-        opts = ExtendedSimpleOption(options={}, go_args=[])
```

Comment:
this was added in error, did not work or fix the problem, so I reverted it again.
see repeated failed attempts to fix the tests https://github.com/hpcugent/vsc-utils/pull/32
 and https://github.com/hpcugent/vsc-utils/pull/33